src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -393,13 +393,13 @@ julia> matrix(ZZ, hilbert_basis(C))
 """
 function hilbert_basis(C::Cone)
```

Comment:
```suggestion
function hilbert_basis(C::Cone{fmpq})
```
Hilbert basis can only be computed for rational cones.